README.md

```diff
@@ -0,0 +1,21 @@
+
+[<p align="center"><img src="./assets/OwlUI_wlogo.png" width="600"/></p>](./OwlUI_wlogo.png)
+
+
+1. [A Grid](#A-Grid)
+
+A Grid
+---
+
+Uma grid de 12 colunas desenvolvida para o OwlUI, uma biblioteca de components totalmente gratuita e Open-Source. 
+Baseada nas propriedades do flexbox, possuí seu fluxo unidirecional de forma extremamente fácil e simples de se configurar e utilizar para criação de sites e aplicações responsivas.
+
+#### Por que usar?
+
+A grid possibilita a criação rápida e tranquila de aplicações responsivas justamente por trazer todas as funcionalidades necessárias, como por exemplo, todas as `@media` queries configuradas para todos os tamanhos de dispositivos disponiveis   
+
+#### Por que resposivo?
+
+Nós queremos que nossos componentes alcancem todos os publicos, e que consiga ser comportada pelo máximo de dispositivos possiveis não importando a orientação da tela ou o seu tamanho. 
+
+
```

Comment:
Agreed, can you make this translation?
One more thing, i guess it would be nice to have a README.pt-br.me for better accessibility like in this [example](https://github.com/tiimgreen/github-cheat-sheet/blob/master/README.ko.md)

Reply:
Agreed! I will translate the README at another PR. Could you rename it to `README.md`?